### ISSUE TITLE
Adding google login feature.

### ISSUE THREAD

**Issue by Pra-kas** [ascoder1109/soochi_shopping_list_app#1]
Sir @ascoder1109  I am GSSOC'24 contributor. I would like to add google login feature in this application.

**Comment by ascoder1109** [comment]
Yes, sure, you can. Just fork my repo into yours and do note that this project has nothing to do with GSSOC 2024.

**Comment by Pra-kas** [comment]
Ok can I start working on it?

**Comment by ascoder1109** [comment]
Yes, sure.
